lib/slack/bot.ex

```diff
@@ -76,7 +76,7 @@ defmodule Slack.Bot do
   end
 
   @doc false
-  def onconnect(_websocket_request, %{slack: slack, process_state: process_state, bot_handler: bot_handler} = state) do
+  def onconnect(_websocket_request, state = %{slack: slack, process_state: process_state, bot_handler: bot_handler} = state) do
```

Comment:
I was trying to figure out why credo was reporting the other way as inconsistent, and forgot to change it back. Fixing!